source/robot_model/include/robot_model/Exceptions/InvalidJointStateSizeException.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <exception>
+#include <iostream>
+
+namespace RobotModel {
+namespace Exceptions {
+class InvalidJointStateSizeException : public std::runtime_error {
+public:
+  explicit InvalidJointStateSizeException(unsigned int state_nb_joints, int robot_nb_joints)
```

Comment:
be consistent with types - I have seen `size_t`, `int` and `unsigned int` being used. For ids I recommend sticking with `unsigned int` since it is used a lot in the body.

Reply:
Yes I realized that I should avoid size_t because Baptiste uses unsigned int but if I but unsigned int for both arguments, i have to cast one of them. Is that better?